gradle/verification-metadata.xml

```diff
@@ -735,6 +730,12 @@
             <pgp value="c328250fe23a2420814521ec0eb69f76fd171538"/>
          </artifact>
       </component>
+      <component group="io.github.gradle" name="gradle-enterprise-conventions-plugin" version="0.7.6">
```

Comment:
Why did this move and why has it have a `sha256` hash now as well?

Reply:
The `group` for this artifact was changed but metadata was not re-generated and so the declaration was in the wrong location in the file.
The sha256 appeared because this is what we document and use for generation. Though I should have edited the origin to confirm it. Will do.